pkgs/development/libraries/zlog/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, pkgs }:
```

Comment:
Right. I don't even see a use of this variable, so I guess I can remove it completely. Hang on!